python-test/features/steps/login_ui.py

```diff
@@ -0,0 +1,74 @@
+from selenium import webdriver
+from behave import given, when, then
+from test_config import TestConfig
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from hamcrest import *
+
+configs = TestConfig.configs()
+base_orb_url = "https://" + configs.get('orb_address')
+user_email = configs.get('email')
+user_password = configs.get('password')
+
+
+@given("that the user is on the orb page")
+def orb_page(context):
+    """Open the required page on Chrome browser
+    Args:
+        context (behave.runner.Context): object that hold contextual information during the running of tests
+    """
```

Comment:
No need to add docs in step functions. The `@given` should be clear enough so we understand what this function does.

Reply:
ok